### PR TITLE
Create section 20 "Supported Inventory" of the Controller UG

### PR DESCRIPTION
**Pull request by EMcWhinn**
Added assembly and reference modules

Create Section 20 of the Controller user Guide

https://issues.redhat.com/browse/AAP-15831

Affects `titles/controller-user-guide`

### CODE REVIEW

File: downstream/modules/platform/ref-controller-amazon-web-services.adoc [hunk 1]
```diff
@@ -0,0 +1,109 @@
+[id="controller-amazon-web-services"]
+
+= Amazon Web Services EC2
+
+----
```
Comment (ianf77) [inline downstream/modules/platform/ref-controller-amazon-web-services.adoc:5]: Can you add a code block header before each [literal, options="nowrap" subs="+attributes"]

It allows you to include attributes and avoid the copy/paste copying any $ or # signs.

**Review by ianf77 — COMMENTED** [review]
One comment

File: downstream/modules/platform/ref-controller-amazon-web-services.adoc (hunk 1 above)
Comment (EMcWhinn) [reply downstream/modules/platform/ref-controller-amazon-web-services.adoc:6]: Done

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**